docker/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+
+volumes:
+  mysql_data:
+      driver: local
+  realm-export.json:
+
+services:
+  mysql:
+      image: mysql:5.7
+      hostname: mysql
+      volumes:
+        - mysql_data:/var/lib/mysql
+      environment:
+        MYSQL_ROOT_PASSWORD: root
+        MYSQL_DATABASE: keycloak
+        MYSQL_USER: keycloak
+        MYSQL_PASSWORD: password
+  keycloak:
+      image: jboss/keycloak
+      hostname: keycloak.local
+      volumes:
+        - ${PWD}/realm-export.json:/export/realm-export.json
+      environment:
+        DB_VENDOR: MYSQL
+        DB_ADDR: mysql
+        DB_DATABASE: keycloak
+        DB_USER: keycloak
+        DB_PASSWORD: password
+        KEYCLOAK_USER: admin
+        KEYCLOAK_PASSWORD: Pa55w0rd
```

Comment:
I suggest just using admin, as this is for development and no one will remember this one after a while ;-)

Reply:
:+1: